integration_tests/test_staking_cache.py

```
@@ -0,0 +1,868 @@
+    """Setup cronos cluster with different staking cache sizes per node."""
+    from pathlib import Path
+
+    path = tmp_path_factory.mktemp("staking_cache")
+    yield from setup_custom_cronos(
+        path,
+        26650,
+        Path(__file__).parent / "configs/staking_cache.jsonnet",
+    )
+
+
+def get_validator_address(cli, node_index):
```

Ruff (0.14.4)

Unused function argument: `node_index`

(ARG001)